proto/metadata.proto

```diff
@@ -80,6 +80,8 @@ message Metadata {
     bool isis_explicit_level_authentication_config = 13;
     // Device skip isis restart-suppress check if value is true.
     bool isis_restart_suppress_unsupported = 14;
+    // Device does not support the ntp nondefault vrf case.
+    bool ntp_non_default_vrf = 15;
```

Comment:
Change the deviation name to `ntp_non_default_vrf_unsupported` to make it a bit more clear?